tools/swift-format/Sources/Rules/OnlyOneTrailingClosureArgument.swift

```diff
@@ -10,4 +10,22 @@ import SwiftSyntax
 /// - SeeAlso: https://google.github.io/swift#trailing-closures
 public final class OnlyOneTrailingClosureArgument: SyntaxLintRule {
 
+  public override func visit(_ node: FunctionCallExprSyntax) {
+    guard containsClosureArgument(argList: node.argumentList) else { return }
```

Comment:
This function can be removed; it can be replaced with an existing function available on sequences:

```swift
guard node.argumentList.contains { $0 is ClosureExprSyntax } else { return }
```